conductor/server/workflows/decommission_device.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "decommission_device",
+    "description": "Removes device info from all services.",
+    "version": 1,
+    "tasks": [
+        {
+            "name": "delete_device_inventory",
+            "taskReferenceName": "delete_device_inventory",
+            "inputParameters": {
+                "http_request": {
+                    "uri": "http://mender-inventory:8080/api/0.1.0/devices/${workflow.input.device_id}",
+                    "method": "DELETE"
```

Comment:
How about including X-Mender-RequestId in request headers? This would allow us to record that certain workflow and associated requests to individual service are made on behalf of some external request identified by requestid header.